### PR TITLE
Add proper README and update COPYING

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
README is based on https://github.com/ansible-collections/collection_template/.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,89 @@
-[![GitHub Actions CI/CD build status — Collection test suite](https://github.com/ansible-collection-migration/community.crypto/workflows/Collection%20test%20suite/badge.svg?branch=master)](https://github.com/ansible-collection-migration/community.crypto/actions?query=workflow%3A%22Collection%20test%20suite%22)
+# Foo Collection
```
Comment (Shaps) [inline README.md]:
```suggestion
# Crypto Collection
```

File: README.md [hunk 2]
```diff
@@ -1,4 +1,89 @@
...
-Ansible Collection: community.crypto
-=================================================
\ No newline at end of file
+Provides modules for [Ansible](https://www.ansible.com/community) for various cryptographic operations.
+
+<!-- Describe the collection and why a user would want to use it. What does the collection do? -->
+
+## Tested with Ansible
+
+Tested with the currently development version of Ansible.
```
Comment (Shaps) [inline README.md]:
```suggestion
Tested with the current development version of Ansible.
```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,89 @@
...
+
+<!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
+
+## External requirements
+
+The exact requirements for every module is listed in the module documentation. Many modules require a recent enough version of [the Python cryptography library](https://pypi.org/project/cryptography/). See the module documentations for the minimal version supported for each module.
```
Comment (Shaps) [inline README.md]:
```suggestion
The exact requirements for every module are listed in the module documentation. 
Most modules require a recent enough version of [the Python cryptography library](https://pypi.org/project/cryptography/). See the module documentations for the minimal version supported for each module.
```

File: README.md (hunk 1 above)
Comment (felixfontein) [reply README.md]: LOL, I missed that! I accidentally closed the README before saving, and had to do most of it again, and apparently overlooked this part ;-)

Comment (felixfontein) [reply README.md]: I've changed it to `# Ansible Community Crypto Collection`.

**Pull request merged**